webapp/src/components/news-form/NewsForm.vue

```diff
@@ -4,6 +4,7 @@
       <v-row>
         <v-col>
           <v-text-field
+			id="newsTitle"
```

Comment:
Don't use HTML ids as they should be unique on a website. If you render the component twice, you would violate that agreement. You could use attributes or classes.